src/cdk/v2/destinations/bluecore/utils.js

```diff
@@ -167,6 +168,38 @@ const normalizeProductArray = (products) => {
   return finalProductArray;
 };
 
+const mapCustomProperties = (message) => {
+  const customProperties = { properties: {} };
+  const messageType = message.type.toUpperCase();
+  switch (messageType) {
+    case 'IDENTIFY':
+      customProperties.properties.customer = extractCustomFields(
+        message,
+        {},
+        ['traits', 'context.traits'],
+        IDENTIFY_EXCLUSION_LIST,
+      );
+      break;
+    case 'TRACK':
+      customProperties.properties.customer = extractCustomFields(
+        message,
+        {},
+        ['traits', 'context.traits'],
+        IDENTIFY_EXCLUSION_LIST,
+      );
+      customProperties.properties = extractCustomFields(
+        message,
+        {},
+        ['properties'],
+        TRACK_EXCLUSION_LIST,
+      );
```

Comment:
```suggestion
      const customerProperties = extractCustomFields(
        message,
        {},
        ['traits', 'context.traits'],
        IDENTIFY_EXCLUSION_LIST,
      );
      customProperties.properties = extractCustomFields(
        message,
        {},
        ['properties'],
        TRACK_EXCLUSION_LIST,
      );
      customProperties.properties.customer = customerProperties;
```
This seems more appropriate. Can you once validate if this is the expectation ?